core/src/main/java/com/linecorp/armeria/common/metric/CertificateMetrics.java

```diff
@@ -43,33 +57,55 @@ final class CertificateMetrics implements MeterBinder {
         for (X509Certificate certificate : certificates) {
             final String commonName = firstNonNull(CertificateUtil.getCommonName(certificate), "");
 
-            Gauge.builder(meterIdPrefix.name("tls.certificate.validity"), certificate, x509Cert -> {
-                     try {
-                         x509Cert.checkValidity();
-                     } catch (CertificateExpiredException | CertificateNotYetValidException e) {
-                         return 0;
-                     }
-                     return 1;
-                 })
-                 .description("1 if TLS certificate is in validity period, 0 if certificate is not in " +
-                              "validity period")
-                 .tags("common.name", commonName)
-                 .tags(meterIdPrefix.tags())
-                 .register(registry);
+            final Gauge validityMeter =
+                    Gauge.builder(meterIdPrefix.name("tls.certificate.validity"), certificate, x509Cert -> {
+                             try {
+                                 x509Cert.checkValidity();
+                             } catch (CertificateExpiredException | CertificateNotYetValidException e) {
+                                 return 0;
+                             }
+                             return 1;
+                         })
+                         .description(
+                                 "1 if TLS certificate is in validity period, 0 if certificate is not in " +
+                                 "validity period")
+                         .tags("common.name", commonName)
+                         .tags(meterIdPrefix.tags())
+                         .register(registry);
+            meters.add(validityMeter);
 
-            Gauge.builder(meterIdPrefix.name("tls.certificate.validity.days"), certificate, x509Cert -> {
-                     final Duration diff = Duration.between(Instant.now(),
-                                                            x509Cert.getNotAfter().toInstant());
-                     return diff.isNegative() ? -1 : diff.toDays();
-                 })
-                 .description("Duration in days before TLS certificate expires, which becomes -1 " +
-                              "if certificate is expired")
-                 .tags("common.name", commonName)
-                 .tags(meterIdPrefix.tags())
-                 .register(registry);
+            final Gauge validityDaysMeter =
+                    Gauge.builder(meterIdPrefix.name("tls.certificate.validity.days"), certificate,
+                                  x509Cert -> {
+                                      final Instant notAfter = x509Cert.getNotAfter().toInstant();
+                                      final Duration diff =
+                                              Duration.between(Instant.now(), notAfter);
+                                      return diff.isNegative() ? -1 : diff.toDays();
```

Comment:
Question) Would it be more useful (and simpler) if we just returned `diff.toDays` as-is so that users can know how long it has been since a certificate expired?